module-2/assignments/activity-2.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<products>
+  <product>
+    <productName>Dog Shirt Gift Basket</productName>
+    <manufacturer>SJB Pet Boutique</manufacturer>
+    <description>Something for every day of the week</description>
+    <price>US$35.99</price>
+    <price>CAD$44.99</price>
+    <price>&#8364;26.79</price>
+    <productItems>1200, 1201, 1202, 1203, 1204, 1205, 1206</productItems>
+    <message>
+        For further information, please visit <our website> or call us on #905-111-2222 & +1-800-111- 2222
+    </message>
```

Comment:
you cannot use <> and & inside xml element. it should be wrapped into <![CDATA[]]>